src/partials/about.html

```diff
@@ -0,0 +1,40 @@
+<body>
+<section class="about">
+  <container>
```

Comment:
To przeważni jest klasa css nie znacznik semantyczny html możesz zmienić to na div z klasa container.